packages/jazz-run/src/index.ts

```diff
@@ -39,6 +41,78 @@ const accountCommand = Command.make("account").pipe(
   Command.withSubcommands([createAccountCommand]),
 );
 
+const ownerOption = Options.text("owner").pipe(
+  Options.withAlias("o"),
+  Options.withDescription("The owner of the group"),
+  Options.withDefault(process.env.JAZZ_WORKER_ACCOUNT),
+);
+
+const ownerSecretOption = Options.text("secret").pipe(
+  Options.withAlias("s"),
+  Options.withDescription("The secret of the owner"),
+  Options.withDefault(process.env.JAZZ_WORKER_SECRET),
+);
+
+const createGroupCommand = Command.make(
+  "create",
+  {
+    owner: ownerOption,
+    ownerSecret: ownerSecretOption,
+    peer: peerOption,
+  },
+  ({ owner, ownerSecret, peer }) => {
+    return Effect.gen(function* () {
+      if (!owner) {
+        throw new Error("Owner is required");
+      }
+      if (!ownerSecret) {
+        throw new Error("Owner secret is required");
+      }
```

Comment:
How about we make this optional and make it possible to pass the amount of workers?